redbot/cogs/audio/audio.py

```diff
@@ -794,11 +795,13 @@ def check(r, u):
             await self._embed_msg(ctx, "Please enter a name for this playlist.")
 
             def check(m):
-                return m.author == ctx.author
+                return m.author == ctx.author and not m.content.startswith(ctx.prefix)
 
             try:
                 playlist_name_msg = await ctx.bot.wait_for("message", timeout=15.0, check=check)
-                playlist_name = str(playlist_name_msg.content)
+                playlist_name = re.sub(
+                    "[^a-zA-Z0-9]", "", str(playlist_name_msg.content.split(" ")[0])
```

Comment:
This will terminate the line at the first space even when contained in double quotes. This doesn't match up with how you can have a playlist name containing spaces if you don't utilize the `wait` feature.

e.g.

`[p]playlist queue "Test Playlist"`
Saves a playlist called, "Test Playlist"

`[p]playlist queue`
_prompt for the user to enter in a name_
`Test Playlist` or `"Test Playlist"`
New playlist called, "Test"

Reply:
Why don't we get rid of the double quotes rule and make `playlist_name` consume rest?